### PR TITLE
Sound effect example

### PR DESCRIPTION
**Pull request by jaesung-0o0**
# Sound Effect

In this project, you can discover how to enable sound effects for direct call.

```swift
/********************************
 AppDelegate.SoundEffect.swift
********************************/

SendBirdCall.setDirectCallDialingSoundOnWhenSilentMode(isEnabled: true)
        
SendBirdCall.addDirectCallSound("Ringing.mp3", forType: .ringing)
SendBirdCall.addDirectCallSound("Dialing.mp3", forType: .dialing)
SendBirdCall.addDirectCallSound("ConnectionLost.mp3", forType: .reconnecting)
SendBirdCall.addDirectCallSound("ConnectionRestored.mp3", forType: .reconnected)
```
### Summary
Enables / disables dial sound used in DirectCall even when the device is in silent mode. Call this method right after addDirectCallSound(_:forType:).

### Declaration
```swift
static func setDirectCallDialingSoundOnWhenSilentMode(isEnabled: Bool)
```

### code snippet
```swift
SendBirdCall.addDirectCallSound("dialing.mp3", forType: .dialing)
SendBirdCall.setDirectCallDialingSoundOnWhenSilentMode(isEnabled: true) // Will play dial direct call sounds in silent mode
```

### Parameters
| Parameter | Descriptionc |
| --- | --- |
| isEnabled | If it is true, dial sound used in DirectCall will be played in silent mode. |

### Summary
Adds sound used in DirectCall such as ringtone and some sound effects with its file name and bundle.

### Declaration
```swift
static func addDirectCallSound(_ name: String, bundle: Bundle = .main, forType type: SoundType)
```

### code snippet
```swift
SendBirdCall.addDirectCallSound("dialing.mp3", forType: .dialing)
```

### Parameters
| Parameter | Description |
| --- | --- |
| name | The name of your audio file. Please explicit its extension: “dialing.mp3” |
| bundle | The bundle object. The default is main bundle. |
| type | The type of sound. |

Simulator Screen Shot - iPhone 12 - 2021-06-09 at 15 15 53

### CODE REVIEW

**Comment by mininny** [comment]
Please rebase this branch to main 🙏

File: Podfile [hunk 1]
```diff
@@ -25,3 +26,11 @@ target 'ScreenShare' do
     project 'ScreenShare/ScreenShare.xcodeproj'
     pod 'SendBirdCalls'
 end
+<<<<<<< HEAD
```
Comment (mininny) [inline Podfile:29]: Conflict here 🚨

File: SoundEffect/SoundEffect/ViewController.swift [hunk 2]
```diff
@@ -0,0 +1,98 @@
+//
+//  ViewController.swift
```
Comment (mininny) [inline SoundEffect/SoundEffect/ViewController.swift:2]: Please rename to CallViewController.swift

**Review by mininny — APPROVED** [review]
LGTM! 🙏

File: SoundEffect/SoundEffect/ViewController.swift (hunk 2 above)
Comment (jaesung-0o0) [reply SoundEffect/SoundEffect/ViewController.swift]: Thanks!!

File: Podfile (hunk 1 above)
Comment (jaesung-0o0) [reply Podfile]: Thanks!

**Pull request merged**